pkg/daemon/struct.go

```diff
@@ -34,6 +34,9 @@ var (
 	SyncInterval              = "katalog-sync.wish.com/sync-interval"     // How frequently we want to sync this service
 	ConsulServiceCheckTTL     = "katalog-sync.wish.com/service-check-ttl" // TTL for the service checks we put in consul
 	ContainerExclusion        = "katalog-sync.wish.com/container-exclude" // comma-separated list of containers to exclude from ready check
+
+	// Meta keys that are used as per-service flags
+	MetaAlwaysHealthy = "_always_healthy"  // always mark service as healthy in Consul
```

Comment:
Do we want this as metadata on the consul service? Or as an annotation on the pod? Annotations in the pod would only show up in k8s (not in consul) whereas this metadata would show up both places.

Reply:
we don't really need it as metadata on consul though it should be harmless either (can serve as some kind of documentation). The problem of add annotations is we would need add two new annotations: 'service-always-healthy' and 'service-always-healthy-'  and we need to do it again if we need to add another flag. So we are using the metadata as a way to pass flag here. Actually I think a better way is to use just one annotation and each service is a json object (including name, port, tag, meta, whatever flags), but that's out of scope for this PR.

Reply:
If we believe the correct approach is annotations then I think we should add this as an annotation and separately refactor. Instead of adding configuration through consul metadata as well

Reply:
done. Passing a status string instead of a flag here which should be more flexible.